src/main/java/net/dv8tion/jda/api/events/guild/member/update/GuildMemberUpdateAvatarEvent.java

```diff
@@ -70,6 +71,22 @@ public String getOldAvatarUrl() {
         return previous == null ? null : String.format(Member.AVATAR_URL, getMember().getGuild().getId(), getMember().getId(), previous, previous.startsWith("a_") ? "gif" : "png");
     }
 
+    /**
+     * Returns an {@link ImageProxy} for this member's old avatar
+     *
+     * @return Possibly-null {@link ImageProxy} of this member's old avatar
+     *
+     * @see    #getOldAvatarUrl()
+     */
+    @Nullable
+    public ImageProxy getOldAvatar()
```

Comment:
It is very likely that downloading the old avatar will fail, we should probably put a note in the docs for that